topology/probes/k8s/probe.go

```diff
@@ -36,15 +36,16 @@ type Probe struct {
 
 func makeProbeBundle(g *graph.Graph) *probe.ProbeBundle {
 	name2ctor := map[string](func(*graph.Graph) probe.Probe){
-		"cluster":       newClusterProbe,
-		"container":     newContainerProbe,
-		"deployment":    newDeploymentProbe,
-		"ingress":       newIngressProbe,
-		"namespace":     newNamespaceProbe,
-		"networkpolicy": newNetworkPolicyProbe,
-		"node":          newNodeProbe,
-		"pod":           newPodProbe,
-		"service":       newServiceProbe,
+		"cluster":               newClusterProbe,
+		"container":             newContainerProbe,
+		"deployment":            newDeploymentProbe,
+		"ingress":               newIngressProbe,
+		"namespace":             newNamespaceProbe,
+		"networkpolicy":         newNetworkPolicyProbe,
+		"node":                  newNodeProbe,
+		"persistentvolumeclaim": newPersistentVolumeClaimProbe,
+		"pod":     newPodProbe,
```

Comment:
please fix indentation

Reply:
surprisingly if the key is 3 char ("pod") only gofmt re-indent to another way
could you move "pod" on top, for consistency